setuptools/tests/test_easy_install.py

```diff
@@ -890,6 +890,72 @@ def test_setup_requires_with_transitive_extra_dependency(
                 monkeypatch.setenv(str('PIP_TIMEOUT'), str('0'))
                 run_setup(test_setup_py, [str('--version')])
 
+    def test_setup_requires_with_distutils_command_dep(self, monkeypatch):
```

Comment:
Can we get a docstring to describe what behaviors this test is asserting?

Reply:
Actually, I'm not sure there's a point in adding the test, as it's mostly testing the functionality being removed...

Reply:
There's only some partial overlap with `test_setup_requires_with_transitive_extra_dependency`, so better to keep it.